### PR TITLE
[BUG] Double deaths cause `TargetingPlugin.compute_targets` to crash

### PR DESCRIPTION
**Pull request by jsyiek**
image

### CODE REVIEW

File: AU2/plugins/custom_plugins/TargetingPlugin.py [hunk 1]
```diff
@@ -350,6 +350,15 @@ def update_graph(
         The problem is constraint satisfaction so a Prolog-style "generate-and-test" is sufficient.
         """
 
+        deaths_adj = list(set([d for d in deaths if d in targeting_graph]))
+        visited = []
+        for d in deaths:
+            if d not in deaths_adj or d in visited:
+                print(f"WARNING: {d} has been killed more than once. Skipping this player when updating graph...")
```
Comment (NerdyNinja11) [inline AU2/plugins/custom_plugins/TargetingPlugin.py:357]: Do we do print statements? I've been returning Labels for warnings to the umpires (compatibilty for a future html frontend and all that)
Unless you do want this to just be a console warning

**Comment by jsyiek** [comment]
I'm going to add a regression test tomorrow to this PR - but curious for thoughts on the fix

**Review by NerdyNinja11 — APPROVED** [review]
Looks good. Probably just double check with the umpires that this does, in fact, generate the same graph before any emails are sent

File: AU2/plugins/custom_plugins/TargetingPlugin.py (hunk 1 above)
Comment (jsyiek) [reply AU2/plugins/custom_plugins/TargetingPlugin.py:357]: Valid point.

Comment (jsyiek) [reply AU2/plugins/custom_plugins/TargetingPlugin.py:357]: I've got an idea for fixing this. Let's get the fix to prod, and then I'll spin up another PR with some new library functions.

**Comment by jsyiek** [comment]
ah, crap, I committed the tests directly to master. no matter. they'd have gone directly there anyway.

**Pull request merged**